### DISCUSSION TITLE
測試

### DISCUSSION THREAD

**Discussion by obai-gs** [share121/inter-knot#5247] [其他]
測試我到底有沒有登錄

**Comment by TNCNET** [comment]
![Screenshot_2024-09-20-22-32-22-586_com share121 inter_knot](https://github.com/user-attachments/assets/27d862cc-fb02-475c-ac3a-75305c6fb80b)

一般来说你需要确认图片中左上角的情况，是不是显示了用户名；没有的话，设置中选用户接口，然后按提示去GitHub绑定。

**Reply by obai-gs** [reply]
谢谢，解决了！
